src/core/EdFi.DataManagementService.Core.External/Backend/DeleteResult.cs

```diff
@@ -2,7 +2,7 @@
 // Licensed to the Ed-Fi Alliance under one or more agreements.
 // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 // See the LICENSE and NOTICES files in the project root for more information.
-namespace EdFi.DataManagementService.Core.Backend;
+namespace EdFi.DataManagementService.Core.External.Backend;
```

Comment:
I really wanted to interface these Result objects as well, but the C# cleverness that makes them like discriminated unions isn't compatible.